components/calendly_v2/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@pipedream/calendly_v2",
+  "version": "1.0.0",
+  "description": "Pipedream Calendly_v2 Components",
+  "main": "dist/app/calendly_v2.app.mjs",
+  "keywords": [
+    "pipedream",
+    "calendly"
```

Comment:
Does it should say `calendly_v2`?

Reply:
Similar as above, v2 is just to denote on our end that this is the second version of the integration.